runtime.md

```diff
@@ -55,21 +55,23 @@ The lifecycle describes the timeline of events that happen from when a container
    If the runtime is unable to create the environment specified in the [`config.json`](config.md), it MUST [generate an error](#errors).
    While the resources requested in the [`config.json`](config.md) MUST be created, the user-specified program (from [`process`](config.md#process)) MUST NOT be run at this time.
    Any updates to [`config.json`](config.md) after this step MUST NOT affect the container.
-3. Once the container is created additional actions MAY be performed based on the features the runtime chooses to support.
-   However, some actions might only be available based on the current state of the container (e.g. only available while it is started).
-4. Runtime's [`start`](runtime.md#start) command is invoked with the unique identifier of the container.
-5. The [prestart hooks](config.md#prestart) MUST be invoked by the runtime.
-   If any prestart hook fails, the runtime MUST [generate an error](#errors), stop the container, and continue the lifecycle at step 10.
-6. The runtime MUST run the user-specified program, as specified by [`process`](config.md#process).
-7. The [poststart hooks](config.md#poststart) MUST be invoked by the runtime.
+3. Runtime's [`start`](runtime.md#start) command is invoked with the unique identifier of the container.
+4. The [prestart hooks](config.md#prestart) MUST be invoked by the runtime.
+   If any prestart hook fails, the runtime MUST [generate an error](#errors), stop the container, and continue the lifecycle at step 9.
+5. The runtime MUST run the user-specified program, as specified by [`process`](config.md#process).
+6. The [poststart hooks](config.md#poststart) MUST be invoked by the runtime.
```

Comment:
Besides the removal of the old step 3, the rest of this hunk is just renumbering.  Shown more clearly on the command line:

```
$ git show --word-diff origin/pr/732
…
3.[-Once the container is created additional actions MAY be performed based on the features the runtime chooses to support.-]
[-   However, some actions might only be available based on the current state of the container (e.g. only available while it is started).-]
[-4.-] Runtime's [`start`](runtime.md#start) command is invoked with the unique identifier of the container.
[-5.-]{+4.+} The [prestart hooks](config.md#prestart) MUST be invoked by the runtime.
   If any prestart hook fails, the runtime MUST [generate an error](#errors), stop the container, and continue the lifecycle at step [-10.-]
[-6.-]{+9.+}
{+5.+} The runtime MUST run the user-specified program, as specified by [`process`](config.md#process).
[-7.-]{+6.+} The [poststart hooks](config.md#poststart) MUST be invoked by the runtime.
   If any poststart hook fails, the runtime MUST [log a warning](#warnings), but the remaining hooks and lifecycle continue as if the hook had succeeded.
[-8.-]{+7.+} The container process exits.
   This MAY happen due to erroring out, exiting, crashing or the runtime's [`kill`](runtime.md#kill) operation being invoked.
[-9.-]{+8.+} Runtime's [`delete`](runtime.md#delete) command is invoked with the unique identifier of the container.
[-10.-]{+9.+} The container MUST be destroyed by undoing the steps performed during create phase (step 2).
[-11.-]{+10.+} The [poststop hooks](config.md#poststop) MUST be invoked by the runtime.
    If any poststop hook fails, the runtime MUST [log a warning](#warnings), but the remaining hooks and lifecycle continue as if the hook had succeeded.
…
```